dwl.c

```diff
@@ -2699,7 +2719,7 @@ configurex11(struct wl_listener *listener, void *data)
 		return;
 	if (c->isfloating || c->type == X11Unmanaged)
 		resize(c, (struct wlr_box){.x = event->x, .y = event->y,
-				.width = event->width, .height = event->height}, 0);
+			        .width = event->width, .height = event->height}, 0, 1);
```

Comment:
A tab was replaced with spaces.

Reply:
@sevz17: Thank you. I have a hard time convincing emacs to be consistent about the tabs.

The change to `resize(c, c->geom, 0, 1)` works well with no regressions.

I've rebased the branch in my repo with these changes.

Reply:
> @sevz17: Thank you. I have a hard time convincing emacs to be consistent about the tabs.

install an emacs plugin to use editorconfig files and this will help:
```editorconfig
root = true

[*]
end_of_line = lf
insert_final_newline = true
charset = utf-8
trim_trailing_whitespace = true
indent_style = tab
indent_size = 4
max_line_length = 79

[*.md]
indent_style = space
```


> The change to `resize(c, c->geom, 0, 1)` works well with no regressions.
> 
> I've rebased the branch in my repo with these changes.

Great, you can now push the changes in my repo.


Thank you!